### PR TITLE
[Issue-2447] Update chainlist

### PR DESCRIPTION
**Pull request by bluezdot**
### Related issue(s)
#2447 

### Is your feature request related to a problem(s)? Please describe.

- Update new logic for earning

### Describe the solution you'd like

- Temporarily blocking ARCH token transfer.
- Check VARCH balances, cDOT-7/14 balances.
- Check parallel provider (Dwellir).

### Describe alternatives you've considered

- No

### Additional context

- No

### CODE REVIEW

**Comment by saltict** [comment]
🚀 Deployed on https://pr-2448--sw-web-runner.netlify.app

File: packages/extension-base/src/koni/api/dotsama/transfer.ts [hunk 1]
```diff
@@ -206,6 +206,11 @@ export const createTransferExtrinsic = async ({ from, networkKey, substrateApi,
   } else if (_TRANSFER_CHAIN_GROUP.sora_substrate.includes(networkKey) && isTxAssetsSupported) {
     transfer = api.tx.assets.transfer(_getTokenOnChainAssetId(tokenInfo), to, value);
   } else if (isTxBalancesSupported && _isNativeToken(tokenInfo)) {
+    if (networkKey === 'invarch') {
```
Comment (nampc1) [inline packages/extension-base/src/koni/api/dotsama/transfer.ts:209]: Move this to the transfer checking function

**Review by nampc1 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -76,7 +76,7 @@
     "@polkadot/types": "^10.9.1",
     "@polkadot/util": "^12.3.2",
     "@polkadot/util-crypto": "^12.3.2",
-    "@subwallet/chain-list": "0.2.33",
+    "@subwallet/chain-list": "0.2.34-beta.2",
```
Comment (nampc1) [inline package.json:79]: missing DOT logo, check again

File: packages/extension-base/src/koni/api/dotsama/transfer.ts (hunk 1 above)
Comment (nampc1) [reply packages/extension-base/src/koni/api/dotsama/transfer.ts:209]: The checking function is deprecated

**Review by nampc1 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (nampc1) [reply package.json]: fixed in Chainlist v0.2.34-beta.3

**Review by saltict — APPROVED** [review]

**Pull request merged**